test/e2e/common/node/container_probe.go

```diff
@@ -214,7 +214,7 @@ var _ = SIGDescribe("Probing container", func() {
 		Testname: Pod liveness probe, container exec timeout, restart
 		Description: A Pod is created with liveness probe with a Exec action on the Pod. If the liveness probe call does not return within the timeout specified, liveness probe MUST restart the Pod.
 	*/
-	ginkgo.It("should be restarted with an exec liveness probe with timeout [NodeConformance]", func() {
+	ginkgo.It("should be restarted with an exec liveness probe with timeout [MinimumKubeletVersion:1.20] [NodeConformance]", func() {
```

Comment:
should the same apply for the next test?

Reply:
reading https://github.com/kubernetes/kubernetes/pull/94115
my assumption is - yes

Reply:
updated to add the tag to the next test too.